app/controllers/admin/items_controller.rb

```diff
@@ -32,7 +32,7 @@ def update
   private
 
   def valid_params
-    params.require(:item).permit(:name, :description, :price, :overview, :details, :categories, :image, :status)
+    params.require(:item).permit(:name, :description, :price, :overview, :details, :image, :status, { :categories => [] })
```

Comment:
Line is too long. [122/80]<br>Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.